client/state/ui/olark/selectors.js

```diff
@@ -32,3 +33,12 @@ export function isOlarkTimedOut( state ) {
 export function isRequestingOlark( state ) {
 	return state.ui.olark.requesting;
 }
+
+/**
+ * Returns if olark operators are available.
+ * @param   {Object}  state  Global state tree
+ * @returns {Boolean}        true, when olark is ready
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/9261/commits/33185797c2bf3d1b0d5bd8d7b9773db338f98033